### PR TITLE
Footprint

### PR DESCRIPTION
**Pull request by dyusuf**

### CODE REVIEW

File: tools/footprint/footprint.xml [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+            input_dir=\$(python -c "import os; print(os.path.dirname('$bam_file'))")
+
+            &&
+            mv \$(find \$input_dir -name "*.PARAM") PARAM
+
+            &&
+            mv \$(find \$input_dir -name "*.RESULTS") RESULTS
+
+            &&
+            mv \$(find \$input_dir  -name "*.plot2.png") plot2.png
```
Comment (bgruening) [inline tools/footprint/footprint.xml]: I think `mv *.plot2.png plot2.png` should be enough.

**Comment by bgruening** [comment]
@dyusuf very nice work! I have added one small comment.

File: tools/footprint/footprint.xml (hunk 1 above)
Comment (dyusuf) [inline tools/footprint/footprint.xml]: the problem is that the output dir is not the current working dir.

**Comment by dyusuf** [comment]
@bgruening thanks! :)

File: tools/footprint/footprint.xml (hunk 1 above)
Comment (bgruening) [inline tools/footprint/footprint.xml]: `mv $input_dir/*.plot2.png plot2.png` or is this a wired nested folder hierarchy?

Comment (dyusuf) [inline tools/footprint/footprint.xml]: `$input_dir`, cool. this might work. thanks!

Comment (dyusuf) [inline tools/footprint/footprint.xml]: i think I misunderstood just now. I thought there is a default galaxy variable called  input_dir. I guess you mean get rid of `find`, right?

Comment (bgruening) [inline tools/footprint/footprint.xml]: Yes getting rid of `find` would be nice. But if these files are hidden in arbitrary nested folders, it's fine to use it.

Comment (dyusuf) [inline tools/footprint/footprint.xml]: sure. just dit it

File: tools/footprint/footprint.xml [hunk 2]
```diff
@@ -39,16 +39,16 @@
             input_dir=\$(python -c "import os; print(os.path.dirname('$bam_file'))")
 
             &&
-            mv \$(find \$input_dir -name "*.PARAM") PARAM
+            mv \$input_dir/*.PARAM PARAM
```
Comment (bgruening) [inline tools/footprint/footprint.xml]: The leading `\` should be removed. You want to use the cheetah variable, not a bash-ism.

Comment (dyusuf) [inline tools/footprint/footprint.xml]: File "/tmp/tmp170mOf/galaxy-dev/lib/galaxy/tools/evaluation.py", line 454, in build
    raise e
NotFound: cannot find 'input_dir'

Comment (dyusuf) [inline tools/footprint/footprint.xml]: the above message is after removing `\`. input_var is actually a bash variable.

**Comment by bgruening** [comment]
Nice! Thanks!

**Comment by bgruening** [comment]
Awesome! Thanks!

**Pull request merged**